src/utils/Dom.ts

```diff
@@ -58,10 +58,10 @@ export class Dom {
       }
     }
 
-    _.each(children, (child: string | HTMLElement | Dom) => {
+    each(children, (child: string | HTMLElement | Dom) => {
       if (child instanceof HTMLElement) {
         elem.appendChild(child);
-      } else if (_.isString(child)) {
+      } else if (isString(child)) {
         elem.innerHTML += child;
```

Comment:
I thought a better fix would be to escape the string child here. However, we would need to (at the very least( convert all our SVGs from strings to DOM elements. There may also be unintended breaking changes for those using `$$` utility :/